lib/route.js

```diff
@@ -280,6 +257,25 @@ function buildRouting (options) {
         // Must store the 404 Context in 'preReady' because it is only guaranteed to
         // be available after all of the plugins and routes have been loaded.
         fourOhFour.setContext(this, context)
+
+        if (opts.schema) {
+          if (this[kSchemaCompiler] == null && this[kSchemaResolver]) {
+            done(new FST_ERR_SCH_MISSING_COMPILER(opts.method, url))
```

Comment:
@mcollina I moved the step and removed the `throw`, but the `done` is already done at this stage.

Reply:
I think the best answer is to wrap https://github.com/mcollina/avvio/blob/44db3689c70fe8e652eec25e359f72877a4ba918/boot.js#L147 into a try-catch, and forward all those error to error-handling logic in avvio. Would you mind to send a PR there?

Reply:
I will do it